CONTRIBUTING.md

```diff
@@ -139,5 +139,12 @@ You can access the frontend at [localhost:3000](http://localhost:3000). Whatever
     ```
     vagrant ssh
     ```
+13. Changed an email? Test it with mailcatcher:
```

Comment:
nitpick: I'm not sure this need to be in the contribution guide. It's already pretty complicated...